x-pack/plugins/cases/server/client/cases/delete.test.ts

```diff
@@ -95,6 +95,22 @@ describe('delete', () => {
         });
       });
     });
+
+    describe('errors', () => {
+      it(`throws 400 when trying to delete more than ${MAX_DELETE_IDS_LENGTH} files at a time`, async () => {
```

Comment:
This should be testing the `MAX_DELETE_IDS_LENGTH`, right?

Reply:
Yup, do you mean the test name? I left "files" there by mistake 😅

Reply:
Yes, for some reason I thought the `MAX_DELETE_IDS_LENGTH ` was only for the files.